src/faster_traits.rs

```diff
@@ -25,7 +25,9 @@ pub trait FasterValue: Deserialize<'static> + Serialize {
             let _ = sender.send(val);
         }
     }
+}
 
+pub trait FasterRmw: Deserialize<'static> + Serialize {
```

Comment:
Regarding the Deserialize trait. 
https://serde.rs/lifetimes.html

"Note that <T> where T: Deserialize<'static> is never what you want. Also Deserialize<'de> + 'static is never what you want. Generally writing 'static anywhere near Deserialize is a sign of being on the wrong track. Use one of the above bounds instead."

Perhaps we want to use the DeserializeOwned trait instead?

Reply:
Interesting, great catch! Yeah I guess `DeserializeOwned` is what we want to use. I hope it can still be derived though

Reply:
As long as the Derive #[derive(Deserialize)] has been added, it works on DeserializeOwned as well (source: using it in another project :smile: )

Reply:
Haha great, have pushed a change to use this trait